CONTRIBUTING.md

```diff
@@ -144,6 +144,6 @@ Enhancement suggestions are tracked as [GitHub issues](https://guides.github.com
 
 ### JavaScript Styleguide
 
-All JavaScript must adhere to [JavaScript Standard Style](http://standardjs.com/).
+Our JavaScript style is expending the [airbnb](https://github.com/airbnb/javascript) style. You can get more details here: https://github.com/LiskHQ/eslint-config-lisk-base
 
-This Contributing guidelines was inspired by and is based on Atom's contribution guidelines and modified for purposes of this repository: https://github.com/atom/atom/blob/master/CONTRIBUTING.md
\ No newline at end of file
+This Contributing guidelines was inspired by and is based on Atom's contribution guidelines and modified for purposes of this repository: https://github.com/atom/atom/blob/master/CONTRIBUTING.md
```

Comment:
`These contribution guidelines were inspired by and are based on Atom's contribution guidelines. They were modified for the purposes of this repository.`